webpack.config.js

```diff
@@ -16,12 +17,17 @@ var webpackConfig = {
   plugins: [
     new webpack.ContextReplacementPlugin(
       // The (\\|\/) piece accounts for path separators in *nix and Windows
-      /angular(\\|\/)core(\\|\/)src(\\|\/)linker/,
-      path.resolve(__dirname, './src'),
+      /angular(\\|\/)core(\\|\/)@angular/,
```

Comment:
This `ContextReplacementPlugin` has to do with silencing some warnings that webpack outputs. See this issue: https://github.com/angular/angular/issues/11580. The behavior between Angular 2.x and 4.x changed a little, and so with the update to 4.x we also want to update the webpack "workaround". If you are staying on Angular 2.x, you wouldn't want to make this change.

This specific comment in the issue addresses the updated behavior and updated webpack config: https://github.com/angular/angular/issues/11580#issuecomment-282705332